exporter/clickhouseexporter/exporter_logs_test.go

```diff
@@ -101,6 +102,38 @@ func TestExporter_pushLogsData(t *testing.T) {
 	})
 }
 
+func TestLogsExporter_createDatabase(t *testing.T) {
```

Comment:
Could you give more descriptive names to the tests in this function that explain what is being tested in each case? I also think having a case to test when the database name isn't a substring of the DSN would be helpful to make sure every case is covered.

Reply:
done